test/gherkin/parser_test.exs

```diff
@@ -50,6 +50,17 @@ defmodule Gherkin.ParserTest do
     Given there are 1 coffees left in the machine
   """
 
+  @feature_with_multiple_feature_tag """
+  @beverage @coffee
```

Comment:
I noticed this also doesn't support multiple lines of tags like:

```
@beverage
@coffee
Feature: Serve coffee
  ...
```

But that can be addressed later if you're not up to it